### ISSUE TITLE
Remove "shiny" emoji from nicknames

### ISSUE THREAD

**Issue by cillosis** [oliver-ni/poketwo#50]
Currently, pokemon are allowed to be nicknamed as :sparkles: and used to scam unsuspecting members. Suggesting that we filter that from the nickname upon adding or even show an error. 

Could be added at: https://github.com/oliver-ni/poketwo/blob/master/cogs/pokemon.py#L55

I'm not good with python, but some possible code that is along the lines of what I am thinking is:

```lang-python
bannedEmoji = ":sparkles:"
if bannedEmoji in nickname
    return await ctx.send("Nickname cannot contain that emoji.")
```

I am unsure if you would need to support the multiple variants like ":sparkles:" as a string, the literal unicode sparkle emoji, or even server specific emoji id's.

**Comment by oliver-ni** [comment]
I thought about this, but I held off on it for now because it is easy to circumvent by just making a custom emoji that looks the same.

Any other thoughts on ways to stop things like this?

**Comment by Siphlygon** [comment]
Is it possible to have a whitelist of allowed emojis vs a blacklist, or is that abusable too? Otherwise, might be time to remove them from nicknames all together.

**Comment by oliver-ni** [comment]
IMO it's _fairly_ obvious when something is a nickname or not. If there's a way to make nicknames stand out more I'd rather do that than ban emojis.

The image on the embed is shiny as well so I'm not really sure how people fall for it.

**Comment by cillosis** [comment]
It is technically the fault of the buyer for not doing `p!t i [id]` in the trade to confirm, or learn about the scam. From what I have seen, people usually just do the `p!p` output and try to make a deal. From there it is less obvious unless you know to look for the wrapping quotation marks.

Perhaps putting the real shiny emoji **BEFORE** the pokemon name instead of after. Once people learn that a real shiny is prefixed instead of suffixed, then maybe the nickname will be more clear. But again, there will always be people who fall for it anyway.

**Comment by oliver-ni** [comment]
Resolved in 9d7081b85af7ca8dd2907ee9c91f0bf40d63ea0f
